examples/02-HFSS3DLayout/Dcir_in_3DLayout.py

```diff
@@ -0,0 +1,127 @@
+import os
```

Comment:
We need the header in the example to be able to process with Sphinx. you can take it from one of other examples.